src/UserSpaceInstrumentation/OrbitUserSpaceInstrumentation.cpp

```diff
@@ -104,6 +111,13 @@ void StartNewCapture() {
 }
 
 void EntryPayload(uint64_t return_address, uint64_t function_id, uint64_t stack_pointer) {
+  bool& is_in_payload = GetIsInPayload();
+  if (is_in_payload) {
```

Comment:
Small suggestion for an expanded comment (hoping I got it correctly this time):

"We have already overwritten the return address with the address of the exit trampoline. So we need to restore the original return address, which will be in a EntryPayload or ExitPayload or one of their callees."

Reply:
EDITED: will be -> was

Reply:
Thanks, looks good. Replaced the comment.